view/src/main/java/cmm/apps/esmorga/view/registration/RegistrationView.kt

```diff
@@ -0,0 +1,219 @@
+            }
+        }
+    }
+
+    EsmorgaTheme {
+        RegistrationView(
+            uiState = uiState,
+            snackbarHostState = snackbarHostState,
+            onBackClicked = onBackClicked,
+            onRegisterClicked = { name, lastName, email, password, repeatedPassword -> rvm.onRegisterClicked(name, lastName, email, password, repeatedPassword) },
+            validateName = { name -> rvm.validateName(name) },
+            validateLastName = { lastName -> rvm.validateLastName(lastName) },
+            validateEmail = { email -> rvm.validateEmail(email) },
+            validatePass = { password -> rvm.validatePass(password) },
+            validateRepeatedPass = { password, repeatedPassword -> rvm.validateRepeatedPass(password, repeatedPassword) }
+        )
+    }
+}
+
+@Composable
+fun RegistrationView(
+    uiState: RegistrationUiState,
+    snackbarHostState: SnackbarHostState,
+    onBackClicked: () -> Unit,
+    onRegisterClicked: (String, String, String, String, String) -> Unit,
+    validateName: (String) -> Unit,
+    validateLastName: (String) -> Unit,
+    validateEmail: (String) -> Unit,
+    validatePass: (String) -> Unit,
+    validateRepeatedPass: (String, String) -> Unit
+) {
+    var name by remember { mutableStateOf("") }
+    var lastName by remember { mutableStateOf("") }
+    var email by remember { mutableStateOf("") }
+    var password by remember { mutableStateOf("") }
+    var repeatedPassword by remember { mutableStateOf("") }
+    Scaffold(
+        snackbarHost = { SnackbarHost(snackbarHostState) },
+        topBar = {
+            Box(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(
+                        top = 16.dp,
+                        bottom = 16.dp,
+                        start = 16.dp,
+                        end = 8.dp
+                    )
+                    .height(48.dp)
+            ) {
+                Image(
+                    painter = painterResource(id = R.drawable.ic_arrow_back),
+                    contentDescription = stringResource(R.string.event_list_error_title),
+                    modifier = Modifier.align(Alignment.CenterStart).clickable { onBackClicked() }
+                )
+            }
+        }
+    ) { innerPadding ->
+        Column(
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(
+                    top = innerPadding.calculateTopPadding(),
+                    bottom = innerPadding.calculateBottomPadding(),
+                    start = 16.dp,
+                    end = 16.dp
+                )
+                .fillMaxWidth()
+                .verticalScroll(state = rememberScrollState())
+        ) {
+            EsmorgaText(text = stringResource(id = R.string.registration_screen_title), style = EsmorgaTextStyle.HEADING_1, modifier = Modifier.padding(vertical = 16.dp))
```

Comment:
we need to check if we can configure the formatter to avoid one line components and make it consistent